tickets/src/components/content/CreateContent.js

```diff
@@ -89,14 +89,15 @@ export class CreateContent extends Component {
 
   updateStateFromEvent(event) {
     this.setState(state => {
+      const links = event.links ? event.links[0] : null
       return {
         ...state,
         name: event.name,
         lockAddress: event.lockAddress,
         description: event.description,
         date: event.date,
         location: event.location,
-        website: event.links[0].href, // TODO: support multiple links
+        website: links, // TODO: support multiple links
```

Comment:
Do we need to pull the `href` here? You might want to check the length of `event.links`, and only access `href` if there's a link there, since if `links` is an empty array, `links[0]` will return undefined which I think is what happened earlier

Reply:
🤦‍♂ Indeed. Updated!